getting-started/your-first-feature/index.json

```diff
@@ -0,0 +1,61 @@
+{
+  "title": "Add a new feature with UI elements to your DAML app",
+  "description": "Get familiar with the process of adding a new full stack feature to your DAML app",
+  "difficulty": "Beginner",
+  "time": "20 minutes",
```

Comment:
I think it's more like 30 minutes if you're completely unfamiliar with react and DAML.